### PR TITLE
SSZ Readme Update

### LINKED ISSUE #99: Add document for Simple Serialization (SSZ)
### What's wrong
No specification for SSZ for auditing or verification.

### How can it be fixed
Add document on [ssz/README.md](https://github.com/ethereum/beacon_chain/blob/master/ssz/README.md)

### PR DESCRIPTION
**Pull request by Poseyy**
Update to the simple serialization readme. It will need to be updated when the common document is finished with some various information as well as a working link to that document's location (wiki or otherwise). It's pretty plain and low technical detail but if I made a mistake let me know! Thanks. 

Closes #99

### CODE REVIEW

File: ssz/README.md [hunk 1]
```diff
@@ -1 +1,67 @@
 # Simple Serialization (SSZ)
```
Comment (djrtwo) [inline ssz/README.md]: Rename "Simple Serialize" here and through doc

File: ssz/README.md [hunk 2]
```diff
@@ -1 +1,67 @@
...
+
+Simple Serialization is a serialization protocol described by Vitalik Buterin with design rationale outlined [here](https://github.com/ethereum/beacon_chain/tree/master/ssz). --update this link to reflect design rationale document when common document is ready--
+
+SSZ To-Do:  
+[X]Move tests to tests/ssz  
+[ ]Add full test coverage for supported types  
+[ ]Refactor ssz.py
```
Comment (djrtwo) [inline ssz/README.md]: link to issue for both of these to dos https://github.com/ethereum/beacon_chain/issues/100

File: ssz/README.md [hunk 3]
```diff
@@ -1 +1,67 @@
...
+[X]Move tests to tests/ssz  
+[ ]Add full test coverage for supported types  
+[ ]Refactor ssz.py  
+
+## Serialization Examples 
+```
+int8: 5 --> b’\x05’
+bytes: b'cow' --> b'\x00\x00\x00\x03cow'
+address: b'\x35'*20 --> b’55555555555555555555’
+hash32: b'\x35'*32 --> b’55555555555555555555555555555555’
```
Comment (djrtwo) [inline ssz/README.md]: consider adding one example of an array type

**Comment by hwwhww** [comment]
LGTM!

**Pull request merged**